rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -103,9 +103,9 @@ remove_ros_arguments(int argc, char const * const argv[])
     throw exceptions::RCLError(base_exc, "");
   }
 
-  std::vector<std::string> return_arguments(nonros_argc);
+  std::vector<std::string> return_arguments(static_cast<std::size_t>(nonros_argc));
```

Comment:
I don't think you need `std::` in front of `size_t` - none of the other code uses it.

Same below.

Reply:
right, removed